crowdaction/main.go

```diff
@@ -122,6 +125,9 @@ func getListCrowdaction(req events.APIGatewayV2HTTPRequest, status string) (even
 			if err != nil {
 				return events.APIGatewayProxyResponse{Body: err.Error(), StatusCode: http.StatusBadRequest}, nil
 			}
+			if crowdaction.PasswordJoin != "" {
+				crowdaction.PasswordJoin = codeRequired
```

Comment:
I'm usually not a big fan of reusing fields for different variables, but I suppose it will do for the MVP.

(The proper way would probably have been to have a model for the internal representation and transform it into a separate API model which contains the (transformed) fields that are public)